### PR TITLE
Fix weird quickdraw wording

### PR DESCRIPTION
**Pull request by SiboVG**
This PR fixes some weird wording of the quickdraw achievement. I edited it to what I believe was actually meant.

### CODE REVIEW

**Comment by SiboVG** [comment]
Oh, sorry, I continued working on my master branch from #69, so the bad hyperlinks fix is also included.

**Comment by YasinDehfuli** [comment]
check all links, and dutch button link is wrong

File: README.de.md [hunk 1]
```diff
@@ -13,7 +13,9 @@
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 [![ru](https://img.shields.io/badge/Lang-Russian-orange.svg)](https://github.com/4xmen/Get-Github-Achievements/blob/master/README.ru.md)
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-[![nl](https://img.shields.io/badge/Lang-Dutch-green.svg)](https://github.com/emreyunusbas/Get-Github-Achievements/blob/main/README.nl.md)
+[![tr](https://img.shields.io/badge/Lang-Turkish-red.svg)](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.tr.md)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![nl](https://img.shields.io/badge/Lang-Dutch-green.svg)](https://github.com/4xmen/Get-Github-Achievements/blob/main/README.nl.md)
```
Comment (YasinDehfuli) [inline README.de.md:18]: You need to change README_nl.md to README.nl.md

File: README_nl.md
Comment (YasinDehfuli) [inline README_nl.md:1]: Change file name to README.nl.md 

your file is wrong (REAME_nl.md)

**Review by YasinDehfuli — COMMENTED** [review]
Please change README_nl.md to README.nl.md

File: README.de.md (hunk 1 above)
Comment (SiboVG) [reply README.de.md:18]: Oh sorry, I missed that!

**Pull request merged**